fanficfare/adapters/adapter_literotica.py

```diff
@@ -353,6 +352,7 @@ def getChapterText(self, url):
         raw_page = self.get_request(url)
         page_soup = self.make_soup(raw_page)
         pages = page_soup.find('div',class_='l_bH')
+        self.story.extendList('eroticatags',[ stripHTML(t) for t in page_soup.select('div#tabpanel-tags a.av_as') ])
```

Comment:
The problem with doing this in `getPageText()` is that the tags collected will be different if you do 'Update Calibre Metadata from Web Site' mode, or (potentially) when updating an existing story.